src/blocks/remote-data-no-result/block.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 3,
+  "name": "remote-data-blocks/no-result",
+  "version": "0.1.0",
+  "usesContext": [ "remote-data-blocks/remoteData" ],
+  "title": "No Remote Data Results",
```

Comment:
Results or result? Should be consistent with block name

Reply:
Kept it as result